### ISSUE TITLE
Possible new linkers on the LHS at Griffith Uni: Carbamates and Ureas 

### ISSUE THREAD

**Issue by Plebellec** [OpenSourceMalaria/Series4#43]
We are two French students in chemistry at _École Nationale de Chimie de Rennes_ and we are currently undertaking our fourth year internship here in Griffith, under the supervision of **Mark Coster**. We will work on OSM targets such as carbamates or ureas. We will present you our project, all opinions and advice are welcome.

My name is **Camille** (@CamilleHauguel) , and I am going to work on the synthesis of differents **Carbamates** using isocyanates, and first I will try with phenyl isocyanate according to the scheme below:

![reactioncamille](https://user-images.githubusercontent.com/39106890/39791117-25f61848-537d-11e8-9414-39f29fa0b710.png)

The plan, after the [first three step](https://github.com/OpenSourceMalaria/OSM_To_Do_List/issues/521), is to convert the chlorine of the trialozopyrazine into a hydroxyl, and then make this hydroxy-compound react with the phenyl isocyanate, certainly using a base or catalyst. For this last step, I found different bases used in the synthesis of carbamates such as [1,8-bis-dimethylaminonaphthalene](https://www.researchgate.net/post/Can_anyone_suggest_a_good_way_to_do_carbamate_formation_with_either_phenyl_hexyl_isocyanates_for_Poly_Aromatic_Alcohols_hindered_hydrocarbons), [DIPEA](https://www.researchgate.net/post/Can_anyone_suggest_a_good_way_to_do_carbamate_formation_with_either_phenyl_hexyl_isocyanates_for_Poly_Aromatic_Alcohols_hindered_hydrocarbons) or [potassium t-butoxide](https://pubs.acs.org/doi/pdf/10.1021/jo01287a046), or some synthesis also use catalyst such as [N-methylimidazole (NMI)](https://pubs.acs.org/doi/pdf/10.1021/ol303424b).

Other different isocyanates could be tested: 4-fluorophenyl isocyanate, 4-bromophenyl isocyanate,  3,4-dichlorophenyl isocyanate and 3-chlorophenyl isocyanate:

![autrsynthese](https://user-images.githubusercontent.com/39106890/39790782-35ec6b8c-537b-11e8-860c-b104fc56a246.PNG)

My name is **Pauline** and I’ll focus on another targets: the **Ureas**. Below is a scheme showing what Mark proposed for me to work on. 

The first three steps are: first synthesize the 2-chloro-6-hydrazinylpyrazine, then this compound reacted with 4-formylbenzonitrile or 4-chlorobenzaldehyde to form an "hydrazone" and finally cyclisation thanks to PIDA. 
After that, we try to obtain the corresponding amine ([see here](https://github.com/OpenSourceMalaria/Series4/wiki/Synthesis-of-the-Reverse-Amide--Linked-Series)) and to react this amine with different isocyanates. 

Several isocyanates will be tested as you can see. I would start with phenyl isocyanate, then use other isocyanates like 3,4-dichlorophenyl isocyanate, 4-fluorophenyl isocyanate, 4-bromophenyl isocyanate and finally 3-fluorophenyl isocyanate.

![reaction](https://user-images.githubusercontent.com/39106890/39790831-8277f930-537b-11e8-9703-97f6a71f2050.png)

**Comment by david1597** [comment]
Hi Pauline and Camille. Welcome to OSM!

@CamilleHaugueli For the conversion of the chlorotrialozopyrazine to the hydroxy, I recently got around 40% yield for this conversion - completely unintentionally, and completely undesirably. See [ELN](https://au-mynotebook.labarchives.com/share/David%2520Smith%2520OSM/MjUzLjV8MzAxNzEvMTk1L1RyZWVOb2RlLzM1MjIwMDU3OXw2NDMuNQ==). 

Best of luck to you both with your projects. These compounds will be an interesting addition to our collection.

**Comment by MFernflower** [comment]
Chloro-core + koh + crown ether in benzene in a sealed vial at rt for a day or two aught to give you a good amount of hydroxy product - you cant have any other chlorine atoms so the para cyano rhs is best choice!!!!

**Comment by mattodd** [comment]
I'm not sure your organic chemistry is legit there, @MFernflower . Not all aromatic chlorines are equally substitute-able. The one on the heterocycle goes quickly (SNAr) presumably because it's easy to put the excess electrons in the intermediate on one of the ring nitrogens. The pendant Ar ring in the northeast would be expected to have inferior conjugation with the core, making such a mechanism disfavoured. Proof is in the pudding, but that would be the paper prediction here and I think it's what has been observed to date?

**Comment by MFernflower** [comment]
You are correct - I have not been brushing up on mechanisms in a bit and I lapsed!!

**Comment by mcoster** [comment]
@mattodd has concisely put forward exactly the arguments I would make. As a side note, it is worthwhile to note that nitrile groups are not always unreactive, and under suitably forcing conditions, I could imagine the -CN substituted compounds possibly reacting with hydroxide. However, much like predicting stock market crashes, I tend to predict 10 of every 3 side reactions in organic chemistry...

**Comment by MFernflower** [comment]
I guess if you really got violent you could end up doing a cn/oh swap - but as we have seen from david's work: sealed vial at RT with chloro-core, koh and crown ether in toluene aught to get you the hydroxyl product you need

**Comment by MFernflower** [comment]
@mcoster I'm a bit curious - What is the purpose of the BrOH and the Pd/C steps in the hydroxy-core synthesis? I would have thought that chloro-core + crown ether + toluene + koh in sealed vial at 70C would give hydroxy core?

**Comment by mcoster** [comment]
BrOH is just a typo - should be BnOH. Yes, the first thing Camille is trying is KOH, 18-crown-6 and toluene. I don't think a sealed tube will be necessary. @david1597 's production of this compound as a side product in 40% yield bodes well for this transformation. The original plan shows reaction with BnOH then hydrog as a backup - the rationale being that if direct reaction with hydroxide (which has slightly different nucleophilic properties to alkoxides) gave poor yields, the reaction with BnOH is well precedented and hydrogenolysis is usually a robust, simple procedure.

**Comment by MFernflower** [comment]
One last thing - Is it at RT or not @mcoster

**Comment by mattodd** [comment]
Hi @Plebellec and @CamilleHauguel do you have links you can provide to your ELNs?
And could one of you insert an entry for the carbamate on the [DCNYS](https://github.com/OpenSourceMalaria/Series4/wiki/Desirable-Compounds-Not-Yet-Synthesised) page underneath the urea, following the pattern of the other entries there? I think the text you need for the picture will be as shown [here](https://github.com/OpenSourceMalaria/TechOps/wiki/How-to-use-Issues), and the png file you need should appear very soon [here](https://github.com/OpenSourceMalaria/Series4/tree/master/Wiki%20Files/DCNYS).

**Comment by mattodd** [comment]
Looks good @Plebellec . I altered the pic a little to say the GRIDD carbamate, to acknowledge the source of the suggestion, but also to change the Cl in the northeast to OCHF2. It doesn't matter too much if there are little differences, but are you wedded to the chlorine on that ring, or could you instead access the OCHF2? The latter is what we've used for a lot of compounds in this series, meaning it's easier to compare the data, is all.

**Comment by mcoster** [comment]
@mattodd The plan is to explore with the p-chloro and p-cyano compounds first. We have 10-20 g of the p-OCHF2 available very soon from our undergrad lab, which @Plebellec and @CamilleHauguel will push forward with. Re: the ELN, I was worried about going with a different ELN system to other groups, but I really like eLabFTW and being GitHub hosted, free, Open Source ELN, it fits well in OSM ethos. It isn't perfect yet for Open Science, but the developer is exceedingly helpful and responsive to suggestions. The main issue to resolve at present is Google/Yahoo/Bing discoverability...

@MFernflower - yes, Pauline and Camille have made excellent progress so far, but I wouldn't like to put any definitive timeframes on reaching new molecules - research always throws up surprises, eg. the recent unexpected regiochemistry highlighted in #39 !

**Comment by mattodd** [comment]
The ELN looks very good, and it's important we experiment with new things. I guess it's not Google-able, right, if there is a sign-in step, even if you can sign in as an OSM guest (nice touch). But we can try that to test it.

**Comment by mcoster** [comment]
Exactly - the software is designed as "closed by default" and the need to sign in stops Google bots at the door. I've [opened an issue at the eLabFTW repo](https://github.com/elabftw/elabftw/issues/718) and I think the dev @NicolasCARPi is looking into it as a feature request
